src/yuzu/applets/controller.cpp

```diff
@@ -460,6 +467,7 @@ void QtControllerSelectorDialog::UpdateControllerIcon(std::size_t player_index)
     const QString stylesheet = [this, player_index] {
         switch (GetControllerTypeFromIndex(emulated_controllers[player_index]->currentIndex(),
                                            player_index)) {
+        case Settings::ControllerType::GameCube:
         case Settings::ControllerType::ProController:
```

Comment:
minor nit, swap the 2
```suggestion
        case Settings::ControllerType::ProController:
        case Settings::ControllerType::GameCube:
```